src/shorah/cli.py

```diff
@@ -121,6 +121,11 @@ def main():
     parent_parser.add_argument("-I", "--ignore_indels", action="store_true", default=False, dest="ignore_indels",
                                help="ignore SNVs adjacent to insertions/deletions\n(legacy behaviour of 'fil', ignore this option if you don't understand)")
 
+    parent_parser.add_argument('-of', '--out_format', type=str, dest='format',
+                               default=['csv', 'VCF'], nargs='+',
+                               choices=['csv', 'VCF'],
```

Comment:
I am afraid the mix of upper and lower case keyword might risk confusing some users.
Is there an easy way to make it case insensitive ?

Reply:
Its defined as choices for the argparser, excluding that its wrongly spelled. Can make both completly upper oder lower case charactesr